Formula/haxe.rb

```diff
@@ -17,7 +17,7 @@ class Haxe < Formula
     url "https://github.com/HaxeFoundation/haxe.git", :branch => "development"
 
     depends_on "aspcud" => :build
-    depends_on "opam" => :build
+    depends_on "opam" => ["2.0", :build]
```

Comment:
Homebrew `opam` version is 2.0.5, so you are guaranteed that `haxe` will build against that. We don't support version-specific dependencies, so remove that change.